internal/csi-common/utils.go

```diff
@@ -290,3 +290,19 @@ func requirePositive(x int64) int64 {
 
 	return 0
 }
+
+// IsBlockMultiNode checks the volume capabilities for BlockMode and MultiNode.
+func IsBlockMultiNode(caps []*csi.VolumeCapability) (bool, bool) {
```

Comment:
@nixpanic possible to add unit testing for this one?

Reply:
@Madhu-1 do you want the unit testing to be added in the same PR, if not this LGTM and I shall approve it.

Reply:
@nixpanic what do you say?

Reply:
sure, I'll add that (all new helpers should have unit-tests!)

Reply:
Done! :dart: